stage1/init/common/pod.go

```diff
@@ -348,6 +380,18 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		opts = append(opts, unit.NewUnitOption("Service", "SyslogIdentifier", filepath.Base(app.Exec[0])))
 	}
 
+	if flavor == "kvm" {
+		capabilities := kvm.GetAppCapabilities(app)
+		capabilities = append(capabilities, stdCapabilities...)
```

Comment:
If we want to have the exact same behavior as the nspawn flavor, we should add these capabilities (standard plus the apps') to [systemd-system.conf#CapabilityBoundingSet=](https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#CapabilityBoundingSet=)